cirq/google/engine/calibration_test.py

```diff
@@ -119,6 +119,14 @@ def test_to_proto():
         invalid_value.to_proto()
 
 
+def test_calibrations_with_string_key():
+    calibration = cg.Calibration(metrics={'metric1': {('alpha',): [0.1]}})
+    assert calibration == cg.Calibration(calibration.to_proto())
```

Comment:
Is this the proto generated by this?
```
metrics: [{
  name: 'metric1'
  targets: ['alpha']
  values: [{double_val: 0.1}]
}]
```
It might be useful for future readers if this is hardcoded in the test.

Reply:
Added.